modules/core/include/opencv2/core.hpp

```diff
@@ -300,19 +300,19 @@ function does not copy src itself but simply constructs the border, for example:
 @endcode
 @note When the source image is a part (ROI) of a bigger image, the function will try to use the
 pixels outside of the ROI to form a border. To disable this feature and always do extrapolation, as
-if src was not a ROI, use borderType | BORDER_ISOLATED.
+if src was not a ROI, use borderType | cv::BORDER_ISOLATED.
```

Comment:
In fact, in this case I added `cv::` to be able to click on the word in the documentation generated by doxygen. But I'm fine to remove it if you want.